content/specific-skill-success-criteria/classes-and-objects/_index.md

```diff
@@ -62,3 +62,56 @@ class Cuboid(Rectangle):
 ### Overriding
 
 Learner should know how to override a function using super and without using super.
+
+
+## Java
+
```

Comment:
Please refer to the Pytho section. Why is the heading structre here so different? By different I mean missing...

If you think subheadings are a bad idea then it would be good to make sure things are consistent. So you would need to remove the Python subheadings.  But personally I think subheadings add a lot of value and consisteny is important.